common/djangoapps/course_modes/admin.py

```diff
@@ -33,6 +33,7 @@ class CourseModeForm(forms.ModelForm):
 
     class Meta(object):  # pylint: disable=missing-docstring
         model = CourseMode
+        fields = '__all__'
```

Comment:
https://docs.djangoproject.com/en/1.8/topics/forms/modelforms/#selecting-the-fields-to-use